app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -312,8 +309,8 @@ def parse_content_libraries(api_client, parser)
         parser.parse_content_library_item(library_item) if library_item
       end
     end
-  rescue VSphereAutomation::ApiError
-    nil
+  rescue VSphereAutomation::ApiError, Timeout::Error => err
```

Comment:
Changed from `Net::OpenTimeout` to `Timeout::Error` which is the base class for both `OpenTimeout` and `ReadTimeout`